modules/ROOT/pages/using-fcct.adoc

```diff
@@ -15,24 +15,22 @@ You can run `fcct` as a container with docker or podman or download it as a stan
 
 This example uses podman, but you can use docker in a similar manner.
 
-In this example, substitute `v0.2.0` with the desired version, as the `latest` tag corresponds with `master` and not with the latest release.
```

Comment:
Nit: maybe add an explicit note that `release` always tracks latest release, and `latest` tracks git development?